glotaran/project/result.py

```diff
@@ -2,60 +2,111 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
+from dataclasses import replace
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import cast
 
 import numpy as np
 import xarray as xr
 from numpy.typing import ArrayLike
 from tabulate import tabulate
 
 from glotaran.deprecation import deprecate
+from glotaran.io import load_dataset
+from glotaran.io import load_parameters
+from glotaran.io import load_scheme
 from glotaran.io import save_result
 from glotaran.model import Model
 from glotaran.parameter import ParameterGroup
+from glotaran.parameter import ParameterHistory
+from glotaran.project.dataclasses import exclude_from_dict_field
+from glotaran.project.dataclasses import file_representation_field
 from glotaran.project.scheme import Scheme
 from glotaran.utils.ipython import MarkdownStr
 
 
+class IncompleteResultError(Exception):
+    """Exception raised im mandator argument to create a result are missing.
```

Comment:
A few typos. Presumably meant was:
"Exception raised if mandatory arguments to create a result are missing"

Reply:
This was not implemented by me... @s-weigand ?

Reply:
Yap this is my fault.

Reply:
Resolved in 2cf0b6de42a321200fbe762f885fe189b9bb4dd8